CMakeLists.txt

```diff
@@ -1684,6 +1685,12 @@ endif()
 #
 option(STATIC_DEPS "Link dependencies statically" OFF)
 
+# Ableton Link
+include(${CMAKE_CURRENT_SOURCE_DIR}/lib/ableton-link/AbletonLinkConfig.cmake)
```

Comment:
Where do we that that from? Is this is Ubuntu aleton-link-dev package? I think we need a find script for cmake to look if the package is installed and use that.

Reply:
Oh there is one: /usr/share/ableton-link/AbletonLinkConfig.cmake but no build library.

Reply:
We don't use git submoduls

Reply:
Ableton Link is a header only library. It has been copied into the `lib` directory in this PR.

Reply:
Ah Ok. Since it is already a debian package, we should at least conditional use the debian package. 

Here al ist If distros that provide the package: https://repology.org/project/abletonlink/versions
https://repology.org/project/ableton-link/versions

Reply:
For other distros, I think we should pull the package instead of adding it to the lib directory.